plugins/compilers/as-ssem/src/main/jflex/ssem.jflex

```diff
@@ -120,6 +121,8 @@ number = \-?[0-9]+
     return token(NUMBER, Token.LITERAL, (byte)(num & 0xFF));
 }
 
+{inputLetter} { /* do nothing, just continue reading */ }
```

Comment:
But this means that if the text won't match to any instruction, it will not be considered as error. It must be solved somehow better. But I can't do it now, I will approve it but it must be fixed later.